clientv3/integration/mirror_test.go

```diff
@@ -69,3 +70,37 @@ func TestMirrorSync(t *testing.T) {
 		t.Fatal("failed to receive update in one second")
 	}
 }
+
+func TestMirrorSyncBase(t *testing.T) {
+	cluster := integration.NewClusterV3(nil, &integration.ClusterConfig{Size: 1})
+	defer cluster.Terminate(nil)
+
+	cli := cluster.Client(0)
+	ctx := context.TODO()
+
+	for i := 0; i < 2000; i++ {
+		if _, err := cli.Put(ctx, fmt.Sprintf("test%d", i), "test"); err != nil {
```

Comment:
probably we need to spin up a bunch of go-routines (50?) to finish the put. Sync put would be slow, each would require a fsync (>10ms on hdd).